test/service/test_jobs.py

```diff
@@ -238,7 +238,7 @@ def test_does_not_create_database_record_if_cannot_start(self):
         self.assertEqual(0, self.mock_insert_job_user.call_count)
 
     def test_logs_creation_success(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.